inference-engine/src/transformations/src/transformations/op_conversions/einsum_decomposition.cpp

```diff
@@ -0,0 +1,643 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/einsum_decomposition.hpp"
+
+#include <memory>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "itt.hpp"
+
+namespace {
+/// \brief      Check if the transformation is applicable to Einsum with a given equation
+///
+/// \param      subscript          A subscript to check its format
+///
+/// \return     true - applicable, false - not applicable
+///
+bool is_subscript_applicable(const std::string& subscript) {
+    auto labels = ngraph::op::v7::Einsum::extract_labels(subscript);
```

Comment:
Please use opset7 instead of internal versioning.
```suggestion
    auto labels = ngraph::opset7::Einsum::extract_labels(subscript);
```

Reply:
Done